src/pybind/mgr/dashboard/tests/test_rgw_client.py

```diff
@@ -124,6 +124,28 @@ def test_get_placement_targets_from_realm_zone(self, zonegroup_map, zone_info):
         }
         self.assertEqual(expected_result, instance.get_placement_targets())
 
+    @patch.object(RgwClient, '_get_realms_info')
+    def test_get_realms(self, realms_info):
+        realms_info.side_effect = [
+            {
+                "default_info": "51de8373-bc24-4f74-a9b7-8e9ef4cb71f7",
+                "realms": [
+                    "realm1",
+                    "realm2"
+                ]
+            },
+            {}
+        ]
+
+        instance = RgwClient.admin_instance()
+        expected_result = [
+            "realm1",
+            "realm2"
+                ]
+        self.assertEqual(expected_result, instance.get_realms())
+        expected_result = []
+        self.assertEqual(expected_result, instance.get_realms())
```

Comment:
As expected_results is assigned 2 different lists that are only used once how about this:

```suggestion
        self.assertEqual([ "realm1", "realm2" ], instance.get_realms())
        self.assertEqual([], instance.get_realms())
```

Reply:
Done.